types/ckeditor__ckeditor5-engine/src/view/observer/observer.d.ts

```diff
@@ -1,5 +1,9 @@
+import { Emitter, EmitterMixinDelegateChain } from "@ckeditor/ckeditor5-utils/src/emittermixin";
+import EventInfo from "@ckeditor/ckeditor5-utils/src/eventinfo";
+import { PriorityString } from "@ckeditor/ckeditor5-utils/src/priorities";
 import Document from "../document";
 import View from "../view";
+import DomEventData from "./domeventdata";
 
 export default abstract class Observer {
```

Comment:
Same as before? This `implements Emitter`.

Reply:
```
ERROR: 45:5  expect  TypeScript@4.4 compile error:
Property 'off' in type 'Observer' is not assignable to the same property in base type 'Emitter'.
  Type '<N extends string>(event: N, callback?: ((info: EventInfo<Emitter, N>, domEvt: Event, ...rest: any[]) => void) | undefined) => void' is not assignable to type '(event: string, callback?: ((info: EventInfo<Emitter, "">, data: DomEventData) => void) | undefined) => void'.
    Types of parameters 'callback' and 'callback' are incompatible.
      Types of parameters 'data' and 'domEvt' are incompatible.
        Type 'Event' is not assignable to type 'DomEventData'.
```